etna/transforms/outliers.py

```diff
@@ -30,6 +30,28 @@ def __init__(self, in_column: str):
         self.in_column = in_column
         self.outliers_timestamps: Optional[Dict[str, List[pd.Timestamp]]] = None
 
+    def __save_original_values(self, ts: TSDataset) -> "OutliersTransform":
```

Comment:
```suggestion
    def _save_original_values(self, ts: TSDataset) -> "OutliersTransform":
```

Reply:
that's why https://stackoverflow.com/a/1301369